doc/articles/getting-started-uno-figma.md

```diff
@@ -0,0 +1,27 @@
+# Design and Build Uno Applications With Figma
+It is now possible to use Figma to design and build Uno applications. Designers can use the highly collaborative Figma environment to fine-tune the User Experience (UX) while putting in place building blocks of an application following the Material Design language.
+Using the [Uno Platform Figma Plugin](https://www.figma.com/community/plugin/1045528009520465828/Uno-Platform), it is possible to visualize how the application will render and export actual XAML to use in Visual Studio for the application. This application could be ready to localize, ready for accessibility and can optionally make use of Uno Extensions (Reactive, Navigation...).
+## Designer's Side
+https://www.youtube.com/watch?v=5G73e21yN_w
```

Comment:
there's no voiceover or text in the linked video, I'm not sure how much value it adds in a documentation context? Probably not enough to earn its own section

Reply:
@carldebilly  @duke7553 - the YouTube reference can be removed. We will link to new designer-oriented documentation Sylvain prepared. Hopefully that removes this block.